pkg/spyglass/lenses/buildlog/lens.go

```diff
@@ -97,6 +99,17 @@ func (lens Lens) Header(artifacts []api.Artifact, resourceDir string, config jso
 // It is only used if highlight_regexes is not specified in the lens config.
 var defaultErrRE = regexp.MustCompile(`timed out|ERROR:|(FAIL|Failure \[)\b|panic\b|^E\d{4} \d\d:\d\d:\d\d\.\d\d\d]`)
 
+// defaultSandboxPermissions is the default value for iframe_sandbox_permissions lense config if it is not specified.
+var defaultSandboxPermissions = strings.Join(
+	[]string{
+		"allow-scripts",
+		"allow-top-navigation",
+		"allow-popups",
+		"allow-same-origin",
+	},
+	" ",
+)
```

Comment:
This is defaultSandboxPermissions for the parsedConfig, tho I personally like to work with arrays instead of strings with `foo bar zar`. 

Not sure if you prefer this or the `allow-scripts allow-top-navigation allow-popups allow-same-origin` instead. 

Please let me know, I'll update if you want to change it.

Reply:
Suggestion is to do another PR after this to adjust the defaults to not combine `allow-same-origin` together with `allow-scripts`, but I suppose this needs to be communicated to the users of the prow installation, so they are aware of the ability to configure this pr lense.